keyboards/fc660c/keymaps/via_rgb/fc660c_via_RGB.json

```diff
@@ -0,0 +1,16 @@
+{

```

Comment:
This is a VIA json and doesn't belong in this repo.   Please remove it.

Reply:
Done.